### PR TITLE
#29: fixed #34: add email and ms teams infos

### LINKED ISSUE #34: add email and ms teams notification

### PR DESCRIPTION
**Pull request by JoergZeidler**

### CODE REVIEW

File: README_CICD_INTEGRATION.md [hunk 1]
```diff
@@ -70,7 +70,34 @@ Note 3: If you are running the local docker registry, you might add this to the
...
+
+### Email notification
+
+The concourse ci environment automatically could send an e-mail notification about the current build status.
+Copy the file `ci/email.template.yaml` to `ci/email.yaml` and enter the email server configuration and email addresses.
+For further information how to configure the email notification, see:
+
+<https://github.com/pivotal-cf/email-resource>
+
+The provided pipeline.yaml file demonstrate the usage of email notification.
```
Comment (mlilien) [inline README_CICD_INTEGRATION.md]: demonstrate -> demonstrates

File: README_CICD_INTEGRATION.md [hunk 2]
```diff
@@ -70,7 +70,34 @@ Note 3: If you are running the local docker registry, you might add this to the
 
 **Note: Make sure that you have set up ssh access with your ssh-key in your git repository.**
 
-Fill in your ssh private key and your credentials for the docker registry and S3 server in `credentials.example.yaml` and rename it to `credentials.yaml`.
+Fill in your ssh private key and your credentials for the docker registry and S3 server in `credentials.example.yaml` and rename it to `credentials.yaml`. Note the indentation of the ssh key as in the example file.
+
+### Email notification
+
+The concourse ci environment automatically could send an e-mail notification about the current build status.
```
Comment (mlilien) [inline README_CICD_INTEGRATION.md]: The concourse ci environment can automatically send ...

**Review by mlilien — CHANGES_REQUESTED** [review]

**Comment by siredmar** [comment]
@JoergZeidler i guess putting the settings for `email_smtp_pw` and `email_to` in a seperate and gitignored yaml file would be a good idea to avoid getting your smtp password and receiver list stored in git. Also it would be useful to gitignore the files `email.yaml` and `msteams.yaml`

**Comment by JoergZeidler** [comment]
@siredmar: thx for the hint about the gitignore. Your proposal about extra file for email settings is already described in the readme ci.

**Review by mlilien — APPROVED** [review]

**Pull request merged**